docs/bulkIngest.md

```diff
@@ -18,16 +18,26 @@ limitations under the License.
 
 This is an example of how to bulk ingest data into accumulo using map reduce.
 
+This tutorial uses the following Java classes.
+
+ * [SetupTable.java] - creates the table and some data to ingest
+ * [BulkIngestExample.java] - ingest the data using map reduce
+ * [VerifyIngest.java] - checks that the data was ingested
+ 
+Remember to copy the accumulo-examples-*.jar to Accumulo's 'lib/ext' directory.
```

Comment:
The asterisk seems to have special meaning in Markdown. Might be better to escape it.